<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assignments</class>
 <widget class="QDialog" name="assignments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>8</y>
     <width>881</width>
     <height>509</height>
    </rect>
   </property>
   <property name="title">
    <string>Assignments:</string>
   </property>
   <widget class="QTableView" name="tableView_assignments">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>87</y>
      <width>483</width>
      <height>410</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>28</y>
      <width>159</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>View Assignments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_courses"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_addAssignment">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>86</y>
      <width>290</width>
      <height>335</height>
     </rect>
    </property>
    <property name="title">
     <string>Add New Assignment:</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>101</y>
       <width>16</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_pointsEarned">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_percentage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="textEdit_additional">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>213</y>
       <width>254</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>300</y>
       <width>173</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_saveAssignment">
        <property name="text">
         <string>Save Assignment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>27</y>
       <width>255</width>
       <height>180</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Course Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Assignment Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Assignment Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Points Earned:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Percentage:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Points Possible:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Additional Information:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="comboBox_courses_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_categories"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pointsEarned"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_percentage"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pointsPossible">
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget_2</zorder>
    <zorder>textEdit_additional</zorder>
    <zorder>layoutWidget_3</zorder>
    <zorder>layoutWidget_4</zorder>
    <zorder>groupBox_editAssignment</zorder>
    <zorder>groupBox_editAssignment</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_editAssignment">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>86</y>
      <width>290</width>
      <height>335</height>
     </rect>
    </property>
    <property name="title">
     <string>Edit Assignment:</string>
    </property>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>300</y>
       <width>206</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_saveAssignment_2">
        <property name="text">
         <string>Update Assignment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel_2">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>27</y>
       <width>254</width>
       <height>179</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Course Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Assignment Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Assignment Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Points Earned:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Percentage:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Points Possible:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Additional Information:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLineEdit" name="lineEdit_courseNumber">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name_2">
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_categories_2"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pointsEarned_2"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_percentage_2"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_pointsPossible_2">
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>101</y>
       <width>16</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QCheckBox" name="checkBox_pointsEarned_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_percentage_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="textEdit_additional_2">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>213</y>
       <width>254</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>55</y>
      <width>339</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_deleteAssignment">
       <property name="text">
        <string>Delete Assignment</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addAssignment">
       <property name="text">
        <string>Add Assignment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_editAssignment">
       <property name="text">
        <string>Edit Assignment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_deleteAssignment">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>86</y>
      <width>290</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Confirm Assignment Delete:</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>230</width>
       <height>86</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>CONFIRM DELETE OF ASSIGNMENT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_assignmentDelete">
          <property name="text">
           <string>+assignmentNumber</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>ASSIGNMENT WILL BE PERMANENTLY DELETED</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_courses</tabstop>
  <tabstop>pushButton_editAssignment</tabstop>
  <tabstop>pushButton_deleteAssignment</tabstop>
  <tabstop>tableView_assignments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
